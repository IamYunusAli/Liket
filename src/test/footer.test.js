import { render } from '@testing-library/react';
import Footer from '../components/footer';

test('should be equal to Coin list', () => {
  const tree = render(
    <Footer />,
  );
  expect(tree).toMatchSnapshot();
});
