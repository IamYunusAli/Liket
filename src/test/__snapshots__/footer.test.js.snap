// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should be equal to Coin list 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <footer>
        <h4>
          Built By
           
          <a
            href="https://github.com/IamYunusAli"
          >
            Yunus Ali
          </a>
        </h4>
        <p>
          Design inspiration by
           
          <a
            href="https://www.behance.net/sakwadesignstudio"
          >
            Nelson Sakwa
          </a>
          and Crypto API
           
          <a
            href="https://www.coingecko.com/"
          >
            Coingecko
          </a>
        </p>
      </footer>
    </div>
  </body>,
  "container": <div>
    <footer>
      <h4>
        Built By
         
        <a
          href="https://github.com/IamYunusAli"
        >
          Yunus Ali
        </a>
      </h4>
      <p>
        Design inspiration by
         
        <a
          href="https://www.behance.net/sakwadesignstudio"
        >
          Nelson Sakwa
        </a>
        and Crypto API
         
        <a
          href="https://www.coingecko.com/"
        >
          Coingecko
        </a>
      </p>
    </footer>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
